pandas/tests/frame/test_convert_to.py

```diff
@@ -175,7 +171,108 @@ def test_to_records_index_name(self):
+                                            periods=10,
+                                            freq='S',
+                                            tz='GMT'))
+        df_utc = DataFrame(date_range('2016-01-01',
+                                      periods=10,
+                                      freq='S',
+                                      tz='GMT'),
+                           index=date_range('2016-01-01',
+                                            periods=10,
+                                            freq='S',
+                                            tz='UTC'))
+        dfgmt = df_gmt[:].tz_convert("UTC")
+        tm.assert_numpy_array_equal(df_utc.to_records(), dfgmt.to_records())
+
+    def test_to_records_with_tz_as_equal_frame(self):
+        # GH13937
+
+        dt1 = [datetime.datetime(2011,
+                                 8,
+                                 15,
+                                 8,
+                                 15,
+                                 12,
+                                 i + 1)
+               for i in range(10)]
+        dt2 = [datetime.datetime(2011,
+                                 8,
+                                 15,
+                                 8,
+                                 15,
+                                 12,
+                                 i + 1,
+                                 pytz.UTC)
+               for i in range(10)]
+        dt3 = [pytz.utc.localize(dt1[i]) for i in range(10)]
+
+        df1 = DataFrame(date_range('2016-01-01',
+                                   periods=10,
+                                   freq='S',
+                                   tz='UTC'),
+                        index=dt2)
+        df1 = df1.tz_convert("GMT")
+        df3 = DataFrame(date_range('2016-01-01',
+                                   periods=10,
+                                   freq='S',
+                                   tz='GMT'),
+                        index=dt3)
+
+        tm.assert_numpy_array_equal(df1.to_records(), df3.to_records())
+
+    def test_from_UTC_to_GMT_as_equal_frame(self):
+        # GH13937
+
+        dt1 = [datetime.datetime(2011,
+                                 8,
+                                 15,
+                                 8,
+                                 15,
+                                 12,
+                                 i + 1,
+                                 pytz.timezone("GMT"))
+               for i in range(10)]
+        dt2 = [datetime.datetime(2011,
+                                 8,
+                                 15,
+                                 8,
+                                 15,
+                                 12,
+                                 i + 1,
+                                 pytz.UTC) for i in range(10)]
+
+        df1 = DataFrame(index=dt1)
+        df3 = df1.tz_convert("UTC")
+
+        df4 = DataFrame(index=dt2)
```

Comment:
however the empty frame is good (but instead of a separate tests, you can do this as part of the other one, IOW)
just use a starting from from the first test